ipfshttpclient/encoding.py

```diff
@@ -105,8 +105,9 @@ def __init__(self) -> None:
 		self._decoder2  = json.JSONDecoder()
 		self._lasterror = None  # type: ty.Optional[ValueError]
 	
-	@ty.no_type_check  # It works just fine and I don't want to rewrite it just
-	                   # because mypy doesn't understand…  # noqa: E114, E116
+	# It works just fine and I don't want to rewrite it just because mypy doesn't understand…
+	# noqa: E114, E116
```

Comment:
Can you check if these `noqa` items are still needed after reformatting?

Reply:
Removed `noqa`.